packages/flutter/test/widgets/custom_single_child_layout_test.dart

```diff
@@ -65,6 +66,25 @@ class FixedSizeLayoutDelegate extends SingleChildLayoutDelegate {
   }
 }
 
+class SizeNotifierLayoutDelegate extends SingleChildLayoutDelegate {
```

Comment:
i'm worried that people will interpret this as "it notifies me when the size changes" as opposed to "i will notify it when the size changes".